battenberg/core.py

```diff
@@ -243,4 +250,4 @@ def upgrade(self, checkout='master', extra_context=None, no_input=False):
         self.repo.lookup_branch(TEMPLATE_BRANCH).set_target(commit.hex)
 
         # Let's merge our changes into HEAD
-        self.merge_template_branch(f'Upgraded template \'{template}\'')
+        self.merge_template_branch(f'Upgraded template \'{template}\'', merge_target)
```

Comment:
Do we think there's value in exposing the commit message? Would it make more sense for us to control that internally?

Reply:
Makes sense to me to add this functionality. Added https://zbrt.atl.zillow.net/browse/ZOMLINFRA-426 to cover this.